images/krte/Dockerfile

```diff
@@ -33,7 +34,8 @@ ARG BAZEL_VERSION_ARG=0.23.1
 # - hinting that we are in a docker container
 ENV IMAGE=${IMAGE_ARG} \
     BAZEL_VERSION=${BAZEL_VERSION_ARG} \
-    PATH=/google-cloud-sdk/bin:${PATH} \
+    GOPATH /home/prow/go \
```

Comment:
this requires an `=`
unfortunately nothing validates this in presubmit

Reply:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-test-infra-push-krte/1273674087943639041